nixos/modules/services/misc/recyclarr.nix

```diff
@@ -0,0 +1,141 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
```

Comment:
We're trying to reducer usage of `with lib;` in NixOS.

IMO, having it over `options` and `meta` is fine, but people are pushing to remove it completely (see #340346 and #208242).